src/components/CardModal/CardModal.tsx

```diff
@@ -31,8 +34,11 @@ export const CardModal: React.FC<Props> = ({
 }) => {
   const { dueDate, description, name } = card;
   const dispatch = useAppDispatch();
+  const [startDate, setStartDate] = useState(dueDate);
```

Comment:
why do you need state for date if you have form that contains date value? try to avoid unnecessary states when using form